SRC/NETWORK/sntp.c

```diff
@@ -86,25 +84,29 @@ enum TcpipErrorCodes {
 };
 
 const char* strPresentation=
-    "SNTP time setter for the TCP/IP UNAPI 1.1\r\n"
-    "by Oduvaldo\r\n"
-	"based on SNTP 1.0 by Konamiman\r\n"
+    "SNTP Time Setter v1.2\r\n"
+    "for the TCP/IP UNAPI 1.1\r\n"
     "\r\n";
 
 const char* strUsage=
-    "Usage: sntp <host>|. [<time zone>] [/d] [/v]\r\n"
+    "Usage: sntp [<host>] [/z <time zone>] [/r <retries>] [/d] [/v] [/h]\r\n"
     "\r\n"
     "<host>: Name or IP address of the SNTP time server.\r\n"
-    "        If \".\" is specified, the environment item TIMESERVER will be used.\r\n"
-    "<time zone>: Formatted as [+|-]hh:mm where hh=00-12, mm=00-59\r\n"
+    "        If nothing is specified, the environment item TIMESERVER will be used.\r\n"
+    "/z <time zone>: Formatted as [+|-]hh:mm where hh=00-12, mm=00-59\r\n"
     "    This value will be added or substracted from the received time.\r\n"
     "    The time zone can also be specified in the environment item TIMEZONE.\r\n"
+    "/r <retries>: Number of retries to get information from time server (Default:1)\r\n"
     "/d: Do not change MSX clock, only display the received value\r\n"
-    "/v: Verbose mode\r\n";
+    "/v: Verbose mode\r\n"
+    "/h: This help\r\n"
+    "\r\n"
+    "original by Konamiman\r\n";

```

Comment:
This isn't needed if you implement the change in the presentation string suggested above.